src/main/java/seedu/address/ui/ViewDisplay.java

```diff
@@ -22,21 +22,21 @@ public ViewDisplay() {
      *
      * @param person The person to be displayed.
      */
-    public void displayEntity(Person person) {
+    public void displayEntity(Person person, String displayString) {
         requireNonNull(person);
         this.getChildren().clear();
-        this.getChildren().add(new ViewPersonDisplay(person).getRoot());
+        this.getChildren().add(new ViewPersonDisplay(person, displayString).getRoot());
     }
 
     /**
      * Displays a {@code Company} in the display box.
      *
      * @param company The company to be displayed.
      */
-    public void displayEntity(Company company) {
+    public void displayEntity(Company company, String displayString) {
         requireNonNull(company);
         this.getChildren().clear();
-        this.getChildren().add(new ViewCompanyDisplay(company).getRoot());
+        this.getChildren().add(new ViewCompanyDisplay(company, displayString).getRoot());
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(